siriuspy/siriuspy/pwrsupply/bsmp/commands.py

```diff
@@ -58,14 +63,15 @@ class PSBSMP(_BSMP):
             CONST.V_WFMREF1_IDX),
     }
 
-    def __init__(self, slave_address, entities, pru=None):
+    def __init__(self, slave_address: int, entities: _EntitiesPS, pru: _IOInterface):
         """Init BSMP."""
-        self.pru = pru
+        self.pru: _IOInterface = pru
+        self.entities: _EntitiesPS
```

Comment:
is this annotation necessary ?

Reply:
It is because we are accessing the member `parameters` which is defined at the child class `EntitiesPS` intead of the base `bsmp.Entities`.

Here for example https://github.com/lnls-sirius/dev-packages/pull/701/files#diff-6852f94bce2009b4831beae5cdc36c289191acda8bf2de952278dd167cbcd9f9R192 the linter would assume that entities is of type `bsmp.Entities` instead of `pwrsupply.bsmp.EntitiesPS`.

This is not the ideal solution but will fix eventual static analyser errors ([mypy](https://mypy.readthedocs.io/en/stable/) ).